### DISCUSSION TITLE
Install Problem

### DISCUSSION THREAD

**Discussion by TFWIsaac** [JakyeRU/Larascord#19] [General]
Hey i want to install your package. But i got this error:

 @php artisan package:discover --ansi
syntax error, unexpected '|', expecting variable (T_VARIABLE)

at C:\laragon\www\larascord\vendor\jakyeru\larascord\src\Console\Commands\InstallCommand.php:27

I just installed a new laravel version a few moments ago...

[Screenshot](https://i.imgur.com/X5YBhEv.png)

**Comment by ValentinNeaga** [comment]
Hello @TFWIsaac, which version of PHP and Laravel are you using?

**Comment by TFWIsaac** [comment]
Laravel v8.78.1 (PHP v7.4.19)

**Comment by ValentinNeaga** [comment]
This issue has been fixed in https://github.com/JakyeRU/Larascord/pull/17/commits/65365e26ca799f9525717670d0798db34be8c000. Until the pull request is ready to be merged into main you can update to the current [stable version of PHP](https://www.php.net/downloads.php) (8.1.1) for a workaround.

**Comment by TFWIsaac** [comment]
Ah okay thanks! Maybe can you can help me, i have tried to build a site with your example and i have some problems with that. Maybe you can take a look at that? [Laracast](https://laracasts.com/discuss/channels/laravel/laravel-tall-stack-discord-login-problems)

**Comment by ValentinNeaga** [comment]
I no longer update ([JakyeRU/Laravel-Discord-Authentication](https://github.com/JakyeRU/Laravel-Discord-Authentication)). I just released [Larascord@3.0.0](https://github.com/JakyeRU/Larascord/releases/tag/v3.0.0) which fixes the issue you've had earlier. Please create a new Laravel application and try again.
